src/block_cache/block_chunkcache.c

```diff
@@ -885,6 +885,10 @@ __wt_chunkcache_setup(WT_SESSION_IMPL *session, const char *cfg[])
           &chunkcache->free_bitmap));
     }
 
+    WT_RET(__wt_config_gets(session, cfg, "chunk_cache.flushed_data_cache_insertion", &cval));
+    cval.val != 0 ? F_SET(chunkcache, WT_CHUNK_CACHE_FLUSHED_DATA_INSERTION) :
```

Comment:
This can be simplified - the flags should always be zero'ed initially, so it can just be
```
if (cval.val != 0)
    F_SET(...);
```

Unless the ternary business is common in other config code, in which case we may as well reuse the pattern here.

Reply:
Agreed! Somehow I thought we'd have to clear it after reopening (with a different config), but it would already be cleared.

Reply:
Update [473f1d9](https://github.com/wiredtiger/wiredtiger/pull/9650/commits/473f1d9dcbc2223f3b1ebb8d49c8034b57845e08)